app/src/main/java/com/concordia/mcga/fragments/NavigationFragment.java

```diff
@@ -22,28 +23,84 @@
 import com.google.android.gms.maps.model.MapStyleOptions;
 import com.google.android.gms.maps.model.Marker;
 import com.google.android.gms.maps.model.MarkerOptions;
+
 import java.util.ArrayList;
 
 public class NavigationFragment extends Fragment implements OnMapReadyCallback, OnCameraIdleListener {
 
+    private enum ViewType {
+        INDOOR, OUTDOOR
+    }
+
     private static final MarkerOptions LOYOLA_MARKER = new MarkerOptions().position(Campus.LOYOLA.getMapCoordinates()).title(Campus.LOYOLA.getName());
     private static final MarkerOptions SGW_MARKER = new MarkerOptions().position(Campus.SGW.getMapCoordinates()).title(Campus.SGW.getName());
     private static final float streetLevelZoom = 15f;
-    Campus currentCampus = Campus.SGW;
-    GoogleMap map;
+    private GoogleMap map;
+
+    //State
+    private ViewType viewType;
+    private Campus currentCampus = Campus.SGW;
     private ArrayList<Marker> buildingMarkers = new ArrayList<>();
```

Comment:
This is not part of MCGA-109. The buildingMarkers list is for Taimoor's google maps markers.